### ISSUE TITLE
Suggestion for showcase

### ISSUE THREAD

**Issue by jschnarr** [almende/vis#982]
Thanks for your work on vis.js. It is a crucial piece of my site, Elite Trade Net. ETN is a trading tool for the game Elite: Dangerous. Take a look and if you're interested in adding it to your showcase here is how I would describe it:

[Elite Trade Net](http://etn.io)
This is a tool for finding profitable trade routes in the game [Elite: Dangerous](https://www.elitedangerous.com/). It uses the vis.js network module to display route options. Animation, select events and dynamic coloring enable graph navigation.

![etn](https://cloud.githubusercontent.com/assets/2323398/8176112/bad5b088-13aa-11e5-8110-f7befd35bfc6.png)

**Comment by AlexDM0** [comment]
Hi jschnarr,

Looks cool :) I'll add it next week. 

Thanks for letting us know!

Cheers

**Comment by AlexDM0** [comment]
Hi,

I've just taken a closer look at your network and I would suggest you alter the edge length. It seems like the settled solution is not optimal. You can try to configure the physics a bit to give a better result.

Regards,

Alex

**Comment by AlexDM0** [comment]
After more thorough examination, there is something off about the physics in your system. Did you make any alterations? The length of the edges cannot be altered through the normal ways and I cannot see them in the edge settings, nor is there a mass property in the nodes so I can't put my finger on what's actually happening... 

I'm still planning to put it on the showcases as it seems like a nicely crafted app but it would greatly benefit of an improved layout.

Regards,

Alex

**Comment by jschnarr** [comment]
Edge length is set explicitly for each edge. Edge length is important
because it represents trade time.

Physics are standard. I like the quirky look of the network but if you have
any suggestions for alternate physics modes I'd be happy to experiment.

Joshua

**Comment by AlexDM0** [comment]
I'd recommend to fix one of the nodes (fixed:true) and turn off the central gravity. Additionally try different types of smooth edges, maybe the dynamic ones arent the best fit for this.

Regards

**Comment by AlexDM0** [comment]
to append to this, you want your edges to determine the network. Turn on configure:'physics' and play around a bit

Regards

**Comment by AlexDM0** [comment]
Hi,

Have you had the chance the review the physics for your system? It's been a little busy lately so I'll add it to the showcases once I have time. Alternatively you're free to make a pull req for it :)

Regards,

Alex

**Comment by jschnarr** [comment]
Yes I tried the other solvers and none look as nice as barnesHut. I relaxed
the central gravity slightly. Removing the gravity altogether made the
graph too spread out, but reducing it marginally improved readablity. The
downside is that it now takes longer to stabilize, and fps during
stabilization can be quite bad (especially on iOS). I also increased the
timestep to 0.6 to try and combat this. Not sure how much it helped.

Joshua
